google/resource_container_cluster.go

```diff
@@ -409,6 +424,12 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		cluster.Description = v.(string)
 	}
 
+	if _, ok := d.GetOk("legacy_abac"); ok {
```

Comment:
if the user specifies `false`, this block won't get triggered, right? So we'll never send `false`, if I recall correctly